django/conf/global_settings.py

```diff
@@ -492,6 +492,7 @@
 PASSWORD_HASHERS = [
     'django.contrib.auth.hashers.PBKDF2PasswordHasher',
     'django.contrib.auth.hashers.PBKDF2SHA1PasswordHasher',
+    'django.contrib.auth.hashers.Argon2PasswordHasher',
```

Comment:
I think (hope?) Python packaging is at a point that if Django wanted to make argon2 the default that it could just depend on argon2-cffi instead of bundling. That being said though, the first mandatory dependency being a C based one might be ambitious, and there isn't much reason to worry about having PBKDF2 as a default. PBKDF2 is considered secure and is a perfectly fine password KDF.